spec/pushmi_pullyu/aip/creator_spec.rb

```diff
@@ -87,11 +88,12 @@
          '30604fd556c96aac01abddf1e8b4e0369fc90ba5 '\
          'data/objects/metadata/files_metadata/856444b6-8dd5-4dfa-857d-435e354a2ead/original_file_metadata.n3',
          '9ea739a91eff6ba99e0227e3a909436d1dfd7ca7 '\
-         'data/objects/files/01bb1b09-974d-478b-8826-2c606a447606/theses.jpg',
+         'data/objects/files/01bb1b09-974d-478b-8826-2c606a447606/Archive_Tar_Minitar_FileNameTooLongHjMQh5EggUDWgpsfoXfyNGoEbvcUcN34YPpjknJhu4y8bs3qVMvBXA5A2aFBxY6EiIyxRS.jpg',
```

Comment:
Instead of turning rubocop off, I would just break up this really long string. You can use the backslash to accomplish this.

The backslash, the line-continuation character, to tell Ruby that these are in fact the same statement, spread over multiple lines.

Think of it kinda like concatenation (long_string becomes short_string + short_string), except theirs no calculation/performance hit, as ruby treats these as the exact same thing.

```suggestion
         'data/objects/files/01bb1b09-974d-478b-8826-2c606a447606/'\
         'Archive_Tar_Minitar_FileNameTooLongHjMQh5EggUDWgpsfoXfyNGo'\
         'EbvcUcN34YPpjknJhu4y8bs3qVMvBXA5A2aFBxY6EiIyxRS.jpg',
```